.github/workflows/acceptance-tests.yml

```diff
@@ -189,6 +189,44 @@ jobs:
         if: failure() || cancelled()
         id: failure_debian
         run: echo '::set-output name=FAILURE_TAG_DEBIAN::true'
+  test-npm-releases:
+    name: NPM acceptance testing
+    runs-on: ubuntu-latest
+    continue-on-error: true
+    outputs:
+      FAILURE_TAG_NPM: ${{ steps.failure_npm.outputs.FAILURE_TAG_NPM }}
+    strategy:
+      matrix:
+        node-version: [ 16.x, 14.x, 10.x, 12.x ]
+    steps:
+      - name: Use Node.js ${{ matrix.node-version }}
+        uses: actions/checkout@v2
+        with:
+          node-version: ${{ matrix.node-version }}
+      - name: Node Install - Testing installation of Twilio CLI (1)
+        if: ${{ github.respository != 'twilio/draft-release-branch' }}
+        run: |
+          npm install -g twilio-cli
```

Comment:
[Q] Are these containers always fresh and won't be a possibility of container re-use so a twilio installation isn't already present?

Reply:
There can be, @charan678 this has already been [implemented](https://github.com/twilio/twilio-cli/blob/draft-release-branch/.github/workflows/cli-test-npm.yml#L26) in draft-release-branch, you could add the same step here. Idea is to keep similar workflows in both the branches to avoid any possible confusions in future.

Reply:
I don't see any problem of caching, @shrutiburman Can you elaborate why did you implement this step